dom-manipulation/main.js

```diff
@@ -0,0 +1,44 @@
+var $number = 0;
+var $button = document.querySelector('.hot-button');
+var $click = document.querySelector('.click-count');
+
+$button.addEventListener('click', function (event) {
+  $number++;
+
+  $click.textContent = 'Clicks: ' + $number;
+
+  if ($number < 4) {
+    $button.className = 'hot-button ' + 'cold';
+  } else if ($number < 7) {
+    $button.className = 'hot-button ' + 'cool';
+  } else if ($number < 10) {
+    $button.className = 'hot-button ' + 'tepid';
+  } else if ($number < 13) {
+    $button.className = 'hot-button ' + 'warm';
+  } else if ($number < 16) {
+    $button.className = 'hot-button ' + 'hot';
+  } else {
+    $button.className = 'hot-button ' + 'nuclear';
```

Comment:
Since both of the values you are concatenating are strings, you don't need to concatenate them, you can just use both of them in one string as the value, no concatenation necessary.